code/workspaces/web-app/src/components/app/TopBar.js

```diff
@@ -1,20 +1,65 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import AppBar from '@material-ui/core/AppBar';
-import Toolbar from '@material-ui/core/Toolbar';
+import { withStyles } from '@material-ui/core/styles';
 import UserInfo from './UserIcon';
+import datalabsLogo from '../../assets/images/datalabs-hori.png';
+import { extendSubdomain } from '../../core/getDomainInfo';
+import TopBarButton from './TopBarButton';
+import navBarLinks from '../../constants/navBarLinks';
 
-const TopBar = ({ identity }) => (
-  <AppBar position="fixed">
-    <Toolbar>
+const logoOrigWidth = 280;
```

Comment:
Would it be better to resize the logo image rather than have to calculate the scaling every time?

Reply:
That's a good idea, I'll do that